apps/adoption/src/app/templates/angular/app.module.ts.txt

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
+import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
+import { ClarityModule } from "@clr/angular";
+
+import { CdsModule } from "@cds/angular";
+
+import { AppComponent } from "./app.component";
+
+import '@clr/icons';
```

Comment:
You are right here - we don't need to include the old icons. But because we are running v4 at the moment - Stackblitz is not working well with v5 at the moment - all the changes with the icons are not available at the moment. 

So when we update to v5 packages this line will be reworked to use `core icons`. 

Without this line for example this demo: 

```ts
import { Component } from "@angular/core";

@Component({
  selector: "app-root",
  template: `
    <clr-alert [clrAlertType]="'danger'" [clrAlertAppLevel]="true">
      <clr-alert-item>
        <span class="alert-text">
          Proin varius lacinia tincidunt. In elementum suscipit lectus eget
          hendrerit. Vestibulum rutrum risus ut turpis interdum, ac auctor nisi
          ultrices
        </span>
      </clr-alert-item>
    </clr-alert>
  `
})
export class AppComponent {}
```

Will render like that

![Screenshot 2021-04-22 at 16 51 44](https://user-images.githubusercontent.com/204564/115726050-0b484780-a38b-11eb-8b1d-32bde3c8a2ad.png)

But this is a good point and must be documented when Stackbliz unblocks us.